### ISSUE TITLE
Example for DELETE doesn't need both Min and Max set

### ISSUE THREAD

**Issue by abatishchev** [Azure/oav#789]
Currently generated examples include bothMinimumSet and MaximumSet for all verbs. But those for DELETE appear to be always identical.

At least provide an option to customize the behavior.

**Comment by raych1** [comment]
@abatishchev , do you mean to allow user to specify these two rules or one of them from CLI?

**Comment by abatishchev** [comment]
Yes, it would be very useful to be able to specify whether user need min, max, or both.

Also whether to put the "mode" into the file name. For example, Cadl expects the file name to match the operation name. But include the mode breaks this expectations, as the result I have to rename one resulting file and delete the another.

In case of delete, the specified preference might be ignored, as I don't think the payload matters. Does it?
